Robus/src/transmission.c

```diff
@@ -99,3 +158,37 @@ static uint8_t Transmit_GetLockStatus(void)
     }
     return ctx.tx.lock;
 }
+/******************************************************************************
+ * @brief finish transmit and try to launch a new one
+ * @param None
+ * @return None
+ ******************************************************************************/
+void Transmit_End(void)
+{
+    if (ctx.tx.status == TX_OK)
+    {
+        // A tx_task have been sucessfully transmitted
+        nbrRetry = 0;
+        ctx.tx.collision = false;
+        // Remove the task
+        MsgAlloc_PullMsgFromTxTask();
+        // Try to send something if we need to.
+        Transmit_Process();
+    }
+    else if (ctx.tx.status == TX_NOK)
```

Comment:
I suggest to perform this statement first with a `return` statement  because there is no `Transmit_Process();` and then `if (ctx.tx.status == TX_OK)`  statement with the `Transmit_Process();` outside of the statement which is performed in both cases `ctx.tx.status == TX_OK` and `TX_DISABLE`.